src/components/Hero/index.jsx

```diff
@@ -3,7 +3,7 @@ import Typography from '@material-ui/core/Typography';
 import Card from '@material-ui/core/Card';
 import Header from '../Header';
 import protestImage from '../../images/protest-illustration.svg';
-import ProtestGuide from '../../documents/ProtestGuide.pdf';
+import ProtestGuide from '../../ProtestGuide.pdf';
```

Comment:
Why are you still importing the `ProtestGuide.pdf` and not the others?

Reply:
This was an oversight.